projects/elife.yaml

```diff
@@ -686,6 +687,8 @@ elife-bot:
                     deletion-policy: delete
                 "{instance}-elife-bot-sessions":
                     deletion-policy: delete
+                "{instance}-elife-bot-digests-input":
+                    deletion-policy: delete
```

Comment:
this is needed so that `elife-bot-formula` builds don't leave buckets around after they have completed